### ISSUE TITLE
[FEATURE][GHG]Module: Material - Domestic Solid Waste d

### ISSUE THREAD

**Issue by guilbep** [EPFL-ENAC/unhcr-geneva-tech-hub-app#543]
<b>comment</b>: Explain disposal options: please add an explanation / clarification for the different waste management options. We presume this is otherwise not clear for tool users. Ideally including a picture.<br/><b>actions</b>: Icons have been created to explain the management practices. IT4R will change its visualisation (sub text)<br/><hr/><b>estimated_hours</b>: 3<br/><b>unhcr_comments</b>: null<br/><b>ctobin111</b>: null<br/><b>me</b>: null

**Comment by guilbep** [comment]
@ctobin111 not sure what to do with that issue tbh ?

**Comment by nmontoya03** [comment]
Hi @guilbep @ctobin111. We had discussed some time ago about this issue related to the description of the waste management alternatives. For now, this description is visible only after selecting a certain alternative. You mentioned it would be possible to include something like a subtext related to the title that would show the description without selecting a specific alternative and checking the info icon. Let me know if you think it would still make sense to implement this change or if you have any question. Thank you!

**Comment by ctobin111** [comment]
@guilbep  Agreed, a subtext below each choice of management option would be ideal.  We can provide you the text.

**Comment by guilbep** [comment]
![Image](https://github.com/EPFL-ENAC/unhcr-geneva-tech-hub-app/assets/161889/b642d1ec-237d-4ad2-a913-ce520c122ebd)

**Comment by guilbep** [comment]
done in 844479f fix(ghg): multiple issue fixing #566, #572
